### ISSUE TITLE
Docker image link seems to be broken / unpublished

### ISSUE THREAD

**Issue by carlos-mg89** [consbio/mbtileserver#76]
Hi and thanks for this great project :)

There are 2 things that didn't work for me while following the README:

1. The local installation output was blank (`go get github.com/consbio/mbtileserver`) and while trying to run `mbtileserver` I receive a command not found message
2. The docker image seems to not be working at all. While trying to pull the docker image I receive this:

> docker pull consbio/mbtileserver:latest
> Error response from daemon: pull access denied for consbio/mbtileserver, repository does not exist or may require 'docker login'

Any help would be very appreciated.

**Comment by brendan-ward** [comment]
@carlos-mg89 sorry for the issues

1. what platform are you on?  This worked fine for me a couple days ago, within a clean Docker container.

2.  I published this in anticipation of getting it hooked up on Docker Hub at the same time.  Unfortunately, we ran into some issues there.  We're working on it and hope to have it resolved in the next couple of days.

**Comment by carlos-mg89** [comment]
Thanks for getting back to me so fast!

1. I have an Ubuntu 18.04
2. That would be wonderful then!

**Comment by brendan-ward** [comment]
@carlos-mg89 I forgot to ask - what version fo Go are you running?

I suspect there might be an issue in the default version of Go for Ubuntu 18.04 (Go 1.10) and our use of modules.  I'm spinning up a Docker container with 18.04 to test that now.

If so, the solution is to use a newer version of Go ([instructions here](https://github.com/golang/go/wiki/Ubuntu)) - I recommend 1.12.  Apologies if the README led you astray here.

**Comment by brendan-ward** [comment]
@carlos-mg89 It worked fine for me in 18.04 Ubuntu in a Docker container.

I think the issue is possibly twofold:
- need to set `$GOPATH` ([instructions here](https://github.com/golang/go/wiki/SettingGOPATH))
- need to make sure that `$GOPATH`/bin is on your system `PATH`

I found that by default it put the binary from `go get ...` in `/root/go/bin`.

You could also try `go get -v -u github.com/consbio/mbtileserver` for more verbose logging, if the above doesn't resolve the issue.

**Comment by carlos-mg89** [comment]
I have go1.10.4.

Apparently the issue is with the $GOPATH variable. I've tried to set it as you pointed out in the link, but it's always empty.

However, I've sorted it out by manually changing directory to /home/my_user/go/bin and from there I can see the mbtileserver binary and I have been able to run it after putting an mbtiles file in the tilesets directory.

Running `go get -v -u github.com/consbio/mbtileserver` just displays that the mbtileserver package was downloaded. nothing else :S
